tools/scripts/download_files.py

```diff
@@ -0,0 +1,56 @@
+import sys
+from bs4 import BeautifulSoup
+import json
+import os
+import requests
+
+NOOS_PATH = sys.argv[1]
+BUILD_PATH = sys.argv[2]
+SERVER_BASE_PATH = sys.argv[3]
```

Comment:
SERVER_BASE_PATH is actually the SERVER BASE PATH plus the HDL folder. 
You can rename it accordingly and pay attention on the value given in azure - it should contain also 'hdl' folder (and most probably the branch name).